vault_core/src/vault.rs

```diff
@@ -333,8 +334,13 @@ impl Vault {
         u16::from(self.program_fee_bps)
     }
 
-    pub fn set_program_fee_bps(&mut self, program_fee_bps: u16) {
+    pub fn set_program_fee_bps(&mut self, program_fee_bps: u16) -> Result<(), ProgramError> {
+        if program_fee_bps > MAX_FEE_BPS {
```

Comment:
there's MAX_BPS and MAX_FEE_BPS. should just use one?

Reply:
We should